<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkDlg</class>
 <widget class="QDialog" name="LinkDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Edit Link</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="QLineEdit" name="lineLink"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushOpen">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushPaste">
     <property name="text">
      <string>Paste</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushSelect">
     <property name="text">
      <string>Select...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>187</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
